resources/jce.rb

```diff
@@ -49,7 +49,8 @@
   jre_path = node['java']['install_type'] == 'jdk' ? 'jre' : ''
 
   if node['os'] == 'windows'
-    include_recipe 'windows'
```

Comment:
This can actually go away entirely. The windows cookbook has nothing in the default recipe

Reply:
Gotcha, yeah @majormoses and I weren't sure on if it was actually needed. Thanks